README.md

```diff
@@ -49,10 +56,10 @@ The Falcon Helm chart has been tested to deploy on the following Kubernetes dist
 # Deployment Considerations
 
 To ensure a successful deployment, you will want to ensure that:
-1. By default, the Helm Chart installs in the `default` namespace. Best
+1. By default when deploying using a Helm Chart, the Helm Chart installs in the `default` namespace. Best
    practices for deploying to Kubernetes is to create a new namespace.
    This can be done by adding `-n falcon-system --create-namespace` to your
-   `helm install` command.
```

Comment:
Should the helm charts create a namespace, eg ``crowdstrike``, if it's best practice not to use ``default``?

Reply:
You have to purposefully create it through a cli flag with helm. It's not recommended or supported through helm to create a namespace in a template.

Reply:
It's ok to run `helm --create namespace` but not do it inside a helm template? O_o

Should we have creation of the namespace be a required step earlier in the process?

Reply:
(meaning certainly believe you, just seems weird!)